libs/interfaces/src/lib/pricing/discount-type.enum.ts

```diff
@@ -1,4 +1,5 @@
 export enum DiscountType {
   percent = 'PERCENT',
   amount = 'AMOUNT',
+  future = 'FUTURE',
```

Comment:
This is not a discount type. Discount type can only be percent (%) and amount (dkk). Should be removed.